src/sqs/index.ts

```diff
@@ -346,13 +346,15 @@ export class SqsQueue<MessagePayload> {
 		} catch (err) {
 
 			statsd.increment(sqsQueueMetrics.failed, this.metricsTags)
-			log.error({err}, "error executing sqs message")
+			log.error({err}, "Error while executing SQS message")
 
 			try {
 				const errorHandlingResult = await this.errorHandler(err, context);
-				if (!errorHandlingResult.retryable ||
-					(errorHandlingResult.skipDlq && context.receiveCount >= this.maxAttempts)) {
-					log.info("Deleting the message hence it reached the maximum amount of retries")
```

Comment:
Noticed we were logging this error for non-retryable errors, which is quite misleading